mirgecom/transport.py

```diff
@@ -698,3 +698,87 @@ def species_diffusivity(self, cv: ConservedVars,
                             eos: Optional[GasEOS] = None) -> DOFArray:
         r"""Get the vector of species diffusivities, ${d}_{\alpha}$."""
         return self._physical_transport.species_diffusivity(cv, dv, eos)
+
+
+class PorousWallTransport(TransportModel):
```

Comment:
I think this can't (or at least shouldn't) inherit from `TransportModel`, because it doesn't implement the same interface (the methods take different arguments). I suspect we could generalize the transport model interface later on to make it compatible. Maybe for now add a `# FIXME: Generalize TransportModel interface to accept state variables other than fluid cv`.

Reply:
Done. No more inheritance and FIXME added.